Progress.js

```diff
@@ -0,0 +1,13 @@
+const Progress = (props) => {
```

Comment:
Hey Alex, nice work here, but it looks like none of your tests are passing. One thing I noticed, is that you're not passing anything called 'amount' or 'target' as props to your Progress component. The one thing you passed to the Progress component in App is 'max={targetAmount}'. So, in order to use that targetAmount, you would need to call 'props.targetAmount'. Also, the Raised amount should be dynamic - how would you calculate the total amount of donations made?